README.md

```diff
@@ -1,4 +1,4 @@
-# Zenithdb Kubernetes Helm Charts
+# Neondatabase Kubernetes Helm Charts
```

Comment:
Shouldn't this be just Neon? Copied from `main`.

Reply:
Yeah I think just "Neon" is best

Reply:
Done.

I replaced two instances.